### ISSUE TITLE
[A11Y]: Accessibility audit round 3 issues

### ISSUE THREAD

**Issue by davidmenendez** [carbon-design-system/carbon-ai-chat#1221]
### High severity

**SL No: 26 - Layout options under chat configuration functionality unclear [demo]**

> Several controls in the Chat Configuration (Show/Hide panel) do not produce any meaningful action when activated.

> "When controls are selected or changed:
> It is unclear whether the controls are working or not implemented.

This may be a nonissue as the options for rounded and square corners are purely visual. We may want to get additional clarification about this.

`Persist Feedback` is also specifically called out. We probably just need to update the label to be more explicit about what this does.

### Medium severity

**SL No: 24. + 27 + 28 - Chat configuration option values unclear [demo]**

> Combo box controls use unclear option labels (Default, True, False), making it difficult to understand their purpose and behavior.

> "In the Chat Configuration section (Show/Hide panel), multiple combo boxes are provided for customizing chat behavior (e.g., “Show input field”). These controls use options such as “Default”, “True”, and “False”. This creates confusion because:
> 1. Labels like “True/False” are technical and do not clearly describe the actual UI changes 
> 2. In many cases, “Default” behaves the same as “True”,  Screen reader users cannot easily understand what selecting each option will do., very difficult  to do configuration ."

We just need to update the values to be more user friendly

> Use clear and descriptive  labels, Label should explain the functional purpose of the controls:
> Replace technical values (“True/False/Default”) with meaningful, user-friendly labels.
> If a default behavior is required, explicitly describe it: Example: “Use System Default (Chat input visible)”
> Ensure each combo box has a clear accessible label describing its purpose.
> Maintain consistency across all similar controls in the Chat Configuration panel.

**SL No: 25 - Show chat history option does not announce that the chat history has been opened or closed [demo]**

When the `Show chat history` checkbox is checked or unchecked the f6 toggle instructions are read, but it's not clear that the chat history window has actually been toggled.

**SL No: 29 - Helper text in sidebar not being read on input focus [demo]**

The `Show stop button immediately` dropdown has some additional helper text that's not read on input focus, which causes that additional information to be lost to screen reader users. The text needs to be associated with input to ensure that it's read by the screen.
